src/@orb.yml

```diff
@@ -3,10 +3,9 @@ version: 2.1
 description: >
   Test and deploy APEX applications to Salesforce.
 
-# This information will be displayed in the orb registry and is not mandatory.
 display:
-  home_url: "https://www.website.com/docs"
-  source_url: "https://www.github.com/EXAMPLE_ORG/EXAMPLE_PROJECT"
+  source_url: "https://developer.salesforce.com"
```

Comment:
source url should be this repo. 

This should be changed to home_url. add source_url